vendor/bat-native-ledger/src/bat/ledger/internal/sku/sku_order.cc

```diff
@@ -28,6 +28,12 @@ SKUOrder::~SKUOrder() = default;
 void SKUOrder::Create(
     const std::vector<ledger::SKUOrderItem>& items,
     ledger::SKUOrderCallback callback) {
+  if (items.empty()) {
```

Comment:
Are they any side effects if "items" key contains an empty list of `order_items`?